tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -111,90 +157,10 @@ static JSONObject fromResponseContributors(final List<GitStatus.ResponseContribu
                 if (scmTriggerItem == null) {
                     continue;
                 }
-                for (final SCM scm : scmTriggerItem.getSCMs()) {

```

Comment:
Thank you both. Fair concern and I'll revert the deletion and keep those code around.